<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_win</class>
 <widget class="QWidget" name="main_win">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1060</width>
    <height>761</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>东南大学 SARA 射电望远镜</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/image/logo.jpg</normaloff>:/image/logo.jpg</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="locale">
   <locale language="Chinese" country="China"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <family>Noto Sans CJK SC Black</family>
       <pointsize>26</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;东南大学SARA射电望远镜&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="baseSize">
      <size>
       <width>10</width>
       <height>5</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <property name="tabsClosable">
      <bool>false</bool>
     </property>
     <property name="movable">
      <bool>false</bool>
     </property>
     <property name="tabBarAutoHide">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="tab_spectrum">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="whatsThis">
       <string/>
      </property>
      <attribute name="title">
       <string>频谱观测</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_11" stretch="2,2,3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="3,2,4,4,2">
         <item>
          <widget class="QFrame" name="frame_3">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_15">
            <item>
             <widget class="QWidget" name="widget_histogram" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_4"/>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QGroupBox" name="groupbox_cali">
             <property name="title">
              <string>校准选项</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QRadioButton" name="radio_calioff">
                <property name="text">
                 <string>校准关闭</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radio_dynamiccali">
                <property name="text">
                 <string>动态校准</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radio_staticcali">
                <property name="text">
                 <string>文件校准</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="button_save">
             <property name="text">
              <string>保存图像</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <widget class="QGroupBox" name="group_integrationmode">
             <property name="title">
              <string>积分设置</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_8" stretch="0,0,0">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QLabel" name="label_integrationtime">
                  <property name="text">
                   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;积分时间&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSlider" name="slider_integration">
                  <property name="minimum">
                   <number>1</number>
                  </property>
                  <property name="maximum">
                   <number>20</number>
                  </property>
                  <property name="sliderPosition">
                   <number>5</number>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="spinbox_integration">
                  <property name="minimum">
                   <number>1</number>
                  </property>
                  <property name="maximum">
                   <number>20</number>
                  </property>
                  <property name="value">
                   <number>5</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QRadioButton" name="radio_movingaverage">
                <property name="text">
                 <string>滑动平均</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radio_iiraverage">
                <property name="text">
                 <string>IIR 滤波</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="group_rfset">
           <property name="title">
            <string>射频调节</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,5,2">
              <item>
               <widget class="QLabel" name="label_freq">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;频率&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="slider_freq">
                <property name="minimum">
                 <number>1410000000</number>
                </property>
                <property name="maximum">
                 <number>1450000000</number>
                </property>
                <property name="singleStep">
                 <number>100</number>
                </property>
                <property name="pageStep">
                 <number>100</number>
                </property>
                <property name="value">
                 <number>1420400000</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spinbox_freq">
                <property name="minimum">
                 <number>1410000000</number>
                </property>
                <property name="maximum">
                 <number>1450000000</number>
                </property>
                <property name="singleStep">
                 <number>100</number>
                </property>
                <property name="stepType">
                 <enum>QAbstractSpinBox::DefaultStepType</enum>
                </property>
                <property name="value">
                 <number>1420400000</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,5,2">
              <item>
               <widget class="QLabel" name="label_gain">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;射频增益&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="slider_gain">
                <property name="minimum">
                 <number>0</number>
                </property>
                <property name="maximum">
                 <number>21</number>
                </property>
                <property name="singleStep">
                 <number>1</number>
                </property>
                <property name="pageStep">
                 <number>1</number>
                </property>
                <property name="value">
                 <number>10</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spinbox_gain">
                <property name="minimum">
                 <number>0</number>
                </property>
                <property name="maximum">
                 <number>21</number>
                </property>
                <property name="singleStep">
                 <number>1</number>
                </property>
                <property name="stepType">
                 <enum>QAbstractSpinBox::DefaultStepType</enum>
                </property>
                <property name="value">
                 <number>10</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,2,1,2">
              <item>
               <widget class="QLabel" name="label_frecorr">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;频率矫正&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="spinbox_freqcorr">
                <property name="minimum">
                 <double>-10.000000000000000</double>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QCheckBox" name="checkbox_enabledb">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>对数信号</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_5">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_18">
            <item>
             <widget class="QLabel" name="label">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="baseSize">
               <size>
                <width>1</width>
                <height>1</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">border-image: url(:/image/seu-logo-512px.png)</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_13">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="widget_waterfall" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_10">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>9</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_14">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="widget_integration" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_12">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_record">
      <attribute name="title">
       <string>数据记录</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QGroupBox" name="group_record">
         <property name="title">
          <string/>
         </property>
         <widget class="QPushButton" name="pushButton">
          <property name="geometry">
           <rect>
            <x>450</x>
            <y>130</y>
            <width>80</width>
            <height>25</height>
           </rect>
          </property>
          <property name="mouseTracking">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>PushButton</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QGroupBox" name="groupBox">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>331</width>
            <height>241</height>
           </rect>
          </property>
          <property name="title">
           <string>GroupBox</string>
          </property>
          <widget class="QLineEdit" name="lineEdit">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>90</y>
             <width>113</width>
             <height>25</height>
            </rect>
           </property>
          </widget>
          <widget class="QDateTimeEdit" name="dateTimeEdit">
           <property name="geometry">
            <rect>
             <x>90</x>
             <y>150</y>
             <width>194</width>
             <height>26</height>
            </rect>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_4">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_16">
          <item>
           <widget class="QWidget" name="widget" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_17"/>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_calibration">
      <attribute name="title">
       <string>频率校准</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>slider_gain</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinbox_gain</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>942</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>1025</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_freq</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinbox_freq</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>923</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>1025</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinbox_gain</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slider_gain</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1025</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel">
     <x>942</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinbox_freq</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slider_freq</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1025</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>923</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_integration</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinbox_integration</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>586</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>635</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinbox_integration</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slider_integration</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>644</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>503</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
